extensions/host/internal.m

```diff
@@ -557,13 +557,11 @@ static int hs_idleTime(lua_State *L) {
         CFNumberGetValue((CFNumberRef)idle, kCFNumberSInt64Type, &time) ;
     } else {
         IOObjectRelease(ioIterator) ;
-        CFRelease(idle) ;
```

Comment:
I don't think we should be calling `CFRelease()` on `idle` - it was retrieved from an array we own, with a `Get` call, which means we are not responsible for freeing it, just the overall array.

Reply:
That was blindly copied and pasted from the source referenced in #1407; since that line's only reached if an error occurs, I don't think it has actually ever been invoked.  I agree with its removal and suspect it was just an oversight on my part.